regression/pages/lms/__init__.py

```diff
@@ -12,3 +12,5 @@
 LOGIN_BASE_URL = 'https://{}:{}@courses.stage.edx.org'.format(
     BASIC_AUTH_USERNAME, BASIC_AUTH_PASSWORD)
 BASE_URL = os.environ.get('test_url', LOGIN_BASE_URL)
+
+BASE_URL_LMS = 'https://stage.edx.org'
```

Comment:
Ah no you are right. Sorry I saw `stage` but thought I saw `studio`. That's my fault. However, part of my confusion is that `BASE_URL` is not very clear as a standalone variable. Perhaps we need to have `BASE_URL_CMS` and `BASE_URL_LMS`? That would be nice and clear.

That goes beyond the scope of this PR though. Why don't you stick with `BASE_URL_LMS` here, and then put in a separate PR to change `BASE_URL` to `BASE_URL_STUDIO`? That one will take a bit of extra lifting with the testeng jenkins DSL, and we can walk through that, I think.